scripts/make/platform.mk

```diff
@@ -1,39 +1,49 @@
 # Architecture and platform resolving
 
-ifeq ($(PLATFORM),)
-  # `PLATFORM` is not specified, use the default platform for each architecture
+# Install `cargo-axplat` by default if not installed.
+ifeq ($(shell cargo axplat --version 2>/dev/null),)
+  $(info "Installing cargo-axplat...")
```

Comment:
Put it to a Makefile target (like `_axconfig-gen`) to avoid triggering the installation when `cargo-axplat ` is not needed (e.g. `make fmt`)